### PR TITLE
Add option to run `cml ci`

### LINKED ISSUE #58: Add option to run `cml ci`
As per https://github.com/iterative/cml.dev/pull/195/files#r799873687:

```yaml
- uses: iterative/setup-cml@v1
  with:
    # runs `cml ci`
    ci: true  # default: true? false? string options (`ci: --unshallow`)?
```

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Closes #58

### CODE REVIEW

**Comment by 0x2b3bfa0** [comment]
```yaml
on: workflow_dispatch
jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: iterative/setup-cml@cml-ci
        with:
          ci: --unshallow --user-name=0x2b3bfa0
        env:
          GITHUB_TOKEN: ${{ github.token }}
```

**Review by DavidGOrtega — CHANGES_REQUESTED** [review]
Thanks for the effort! 🙏 

Im reluctant to accept the PR yet due to philosophical CML questions that we should debate in our meeting

**Comment by 0x2b3bfa0** [comment]
Yes, sure! 🙏🏼 I was asking myself the same questions

File: action.yml [hunk 1]
```diff
@@ -14,6 +14,10 @@ inputs:
     description: Force install CML if it exists.
     default: false
     required: false
+  ci:
+    description: Run `cml ci` after installing CML.
```
Comment (casperdcl) [inline action.yml]:
```suggestion
    description: Also run `cml ci` (`true`) or flags to forward to `cml ci` (str).
```

I think? i.e. `ci: '--user-email=${{ github.actor }}@users.noreply.github.com'`

**Comment by 0x2b3bfa0** [comment]
🔔 @iterative/cml, can you please refresh our collective memory with regard to the outcome of that meeting? 😅

**Comment by dacbd** [comment]
I believe we agreed that it's a good nice-ity to provide, but not a to-be heavily advertised feature (other than here) since it's GitHub actions-specific feature.

**Review by dacbd — APPROVED** [review]
I'm not a fan of the overloading

**Review by casperdcl — COMMENTED** [review]
probably needs the "inputs" section of the readme updated too?

**Comment by 0x2b3bfa0** [comment]
@iterative/cml, what do we do with this?

**Comment by dacbd** [comment]
Close and revisit if something like this is requested.

**Comment by 0x2b3bfa0** [comment]
See also https://iterativeai.slack.com/archives/C01900GSB4J/p1678389554307949 with @dberenbaum's input.

**Pull request closed without merging**